src/assets/locales/fr.json

```diff
@@ -1,4 +1,25 @@
 {
+  "audio-debug-panel.audio-debug-avatar-title": "Avatar",
+  "audio-debug-panel.audio-debug-global-title": "Globals",
```

Comment:
```suggestion
  "audio-debug-panel.audio-debug-global-title": "Globaux",
```

`globals` is not french

Reply:
Yes, all the new audio-debug-panel part is not currently translated.